integration/v3_grpc_test.go

```diff
@@ -598,9 +598,13 @@ func TestV3StorageQuotaAPI(t *testing.T) {
 
 	key := []byte("abc")
 
+	// wait for server to reboot
+	if _, err := kvc.Range(context.TODO(), &pb.RangeRequest{Key: key}, grpc.FailFast(false)); err != nil {
```

Comment:
this relies on the knowledge that proxy uses etcd.client, which will do range with failfast=true no matter what. Right?